XYZDrivers/src/main/java/com/xyzdrivers/repositories/ClaimsRepo.java

```diff
@@ -16,10 +16,28 @@
  */
 @RequestScoped
 public class ClaimsRepo extends Repo<Claim, Integer> {
-        
+    
+    public ClaimsRepo() {}
+
     @Override
-    public Claim get(Integer id) throws RepositoryException {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    public Claim get(Integer id) {
+        Claim claim = null;
+        try {
+            Object[] result = sqlService.retrieve(Claim.TABLE_NAME, Claim.PRIMARY_KEY, id.toString());
+            LocalDate date = LocalDate.parse(result[2].toString());
+            claim = new Claim(
+                    Integer.parseInt(result[0].toString()),
+                    result[1].toString(),
+                    date,
+                    result[3].toString(),
+                    result[4].toString(),
+                    Float.parseFloat(result[5].toString())
+            );
+        } catch (IllegalArgumentException | SQLException ex) {
+            Logger.getLogger(ClaimsRepo.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
To be consistent throw RepositoryException in the event of an error instead of logging.
If no claim could be found with the given id, return null. (I can't tell if your code does that or not - so just for completeness!)